modules/dr-restoring-cluster-state.adoc

```diff
@@ -144,13 +174,15 @@ starting kube-scheduler-pod.yaml
 static-pod-resources/kube-scheduler-pod-8/kube-scheduler-pod.yaml
 ----
 +
+
+The cluster-restore.sh must show that etcd, kube-apiserver, kube-controller-manager and kube-scheduler pods are stopped and then started at the end of the restore process.
```

Comment:
```suggestion
The `cluster-restore.sh` script must show that etcd, kube-apiserver, kube-controller-manager, and kube-scheduler pods are stopped and then started at the end of the restore process.
```
JUst want to check that etcd, kube-apiserver, kube-controller-manager, and kube-scheduler all refer to "pods"?

Reply:
> Just want to check that etcd, kube-apiserver, kube-controller-manager, and kube-scheduler all refer to "pods"?

All of them are pods, mentioned by the script

Reply:
And it would be best if you write "The `cluster-restore.sh` script output must show..."

Reply:
I'm not sure if you tried to make this update, but it is still showing the original formatting and text (`The cluster-restore.sh must show...`)

Please update this line to something like this:
```
The output of the `cluster-restore.sh` script must show...
```

My understanding is that verbatim pod names should be in backticks [1], so I would suggest the following formatting (this suggestion includes a couple minor grammatical tweaks as well):

```suggestion
The output of the `cluster-restore.sh` script must show that the `etcd`, `kube-apiserver`, `kube-controller-manager`, and `kube-scheduler` pods are stopped and then started at the end of the restore process.
```
[1]
> Use backticks for all non-GUI "system items", including:
> 
> - Inline commands, operations, literal values, variables, parameters, settings, flags, environment variables, user input
> - System term/item, user names, unique or example names for individual API objects/resources (e.g., a pod named mypod), daemon, service, or software package
> - RPM packages
> - File names or directory paths

(source: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting)